components/Activity/ActivityEditor.tsx

```diff
@@ -276,14 +276,14 @@ export class ActivityEditor extends PureComponent<ActivityEditorProps> {
           </Form.Label>
           <Col sm={10}>
             <HTMLEditor
-              defaultValue={detail}
-              onChange={code => (this.detailHTML = code)}
+              defaultValue={this.detailHTML}
+              onChange={code => (this.detailHTML = code === '<br>' ? '' : code)}
             />
             <Form.Control
               hidden
               name="detail"
               required
-              defaultValue={this.detailHTML || detail}
+              defaultValue={this.detailHTML}
```

Comment:
```suggestion
              value={this.detailHTML}
```
默认值只会在初始化的时候赋值一次，不可能随着值的改变而改变。

Reply:
已改。 
但是我debug的时候，会随着 this.detailHTML的值变化而变化， 不理解为什么。